README.md

```diff
@@ -24,31 +24,67 @@ git clone git@github.com:PDFTron/webviewer-salesforce.git
 cd webviewer-salesforce
 ```
 
-3. If you have a paid license key, you can remove the watermark from rendered
+3. Extract `WebViewer.zip`, `cd` to the directory the contents were extracted
```

Comment:
```suggestion
3. [Optional] - If you do not need to upgrade/downgrade to a different version you can skip to step 5. Otherwise, extract `WebViewer.zip`, `cd` to the directory the contents were extracted
```